pkg/operator/vars.go

```diff
@@ -0,0 +1,75 @@
+//
+// Copyright (c) 2012-2018 Red Hat, Inc.
+// This program and the accompanying materials are made
+// available under the terms of the Eclipse Public License 2.0
+// which is available at https://www.eclipse.org/legal/epl-2.0/
+//
+// SPDX-License-Identifier: EPL-2.0
+//
+// Contributors:
+//   Red Hat, Inc. - initial API and implementation
+//
+package operator
+
+import (
+	"github.com/eclipse/che-operator/pkg/util"
+	"strings"
+)
+
+var (
+	// general config
+	cheFlavor           = util.GetEnv("CHE_FLAVOR", "che")
+	namespace           = util.GetNamespace()
+	protocol            = "http"
+	wsprotocol          = "ws"
+	cheHost             string
+	tlsSupport          = util.GetEnvBool("CHE_TLS_SUPPORT", false)
+	pvcStrategy         = util.GetEnv("CHE_INFRA_KUBERNETES_PVC_STRATEGY", "common")
+	pvcClaimSize        = util.GetEnv("CHE_INFRA_KUBERNETES_PVC_QUANTITY", "1Gi")
+	selfSignedCert      = util.GetEnv("CHE_SELF__SIGNED__CERT", "")
+	openshiftOAuth      = util.GetEnvBool("CHE_OPENSHIFT_OAUTH", false)
+	oauthSecret         = util.GeneratePasswd(12)
+	oAuthClientName     = "openshift-identity-provider-" + strings.ToLower(util.GeneratePasswd(4))
```

Comment:
it's a bit strange to see using password as prefix of oAuth client. Maybe it makes sense (maybe not) rename `GeneratePasswd` to `GenerateString`